src/lib/data/html/speeddial.html

```diff
@@ -528,20 +603,26 @@
         $('#fadeOverlay').click();
 });
 
-function emitChanged(pages)
-{
+function searchToggle() {
+    if ("%SEARCH-ENABLED%" == 'true') {
```

Comment:
This can be just

    external.speedDial.setSearchEnabled(%SEARCH-ENABLED%);

Also, move the search enable variable to top as with all other configurable vars: `var SEARCH_ENABLED = %SEARCH-ENABLED%;`, so this line just becomes:


    external.speedDial.setSearchEnabled(SEARCH_ENABLED);

Reply:
It can't be `setSearchEnabled(SEARCH_ENABLED);`
but it can be `setSearchEnabled(!SEARCH_ENABLED);`, this returns value reverse to existing

Changed to:
```
function searchToggle() {
    external.speedDial.setSearchEnabled(!SEARCH_ENABLED);
    location.reload();
}
```

Reply:
Sure